backend/migrations/Version20250831193230.php

```
@@ -0,0 +1,210 @@
+
+/**
+ * Tag System Implementation - Core tables for flexible tagging
+ */
+final class Version20250831193230 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Create tag system tables for flexible tagging of tasks, projects, and files';
+    }
+
+    public function up(Schema $schema): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$schema'. (Unused Code Rules)

(UnusedFormalParameter)